apis/Google.Devtools.AspNet/Google.Devtools.AspNet.Tests/CloudTrace/TraceUtils.cs

```diff
@@ -40,5 +43,26 @@ internal static RateLimiter GetRateLimiter(double qps, long[] elapsedMillisecond
             watch.Setup(w => w.GetElapsedMilliseconds()).Returns(() => returnQueue.Dequeue());
             return new RateLimiter(qps, watch.Object);
         }
+
+        /// <summary>
+        /// Checks if a span's labels are are equal to a dictionary strings.
+        /// </summary>
+        internal static bool IsValidAnnotation(TraceSpan span, Dictionary<string, string> annotation)
```

Comment:
Just as a thought, you might want to make this an extension method. Probably not something to do in this PR.